### PR TITLE
Graphing cleanup

### PR DESCRIPTION
**Pull request by murrant**
Started to fix why some settings weren't applied in embedded email graphs and ended up with a massive cleanup.

After this is merged and confirmed to not break anything, I will seek and destroy the following from the graphing code:

- $_GET
- global variables
- includes/html/graphs/common.inc.php

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: LibreNMS/Data/Graphing/GraphParameters.php [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+        return $options;
+    }
+
+    public function __toString(): string
+    {
+        return implode(' ', $this->toRrdOptions());
+    }
+
+    public function parseAtTime(string|int $time): int
```
Comment (Jellyfrog) [inline LibreNMS/Data/Graphing/GraphParameters.php:196]: This function/logic should probably live in a date/time class?

Comment (murrant) [reply LibreNMS/Data/Graphing/GraphParameters.php:196]: @Jellyfrog sounds good to me

**Comment by Jellyfrog** [comment]
Nice!
It failed as I suspected due to test being done on php7. Think it's rather safe to simply disable it for now

**Comment by fcqpl** [comment]
@murrant looks like this commit broke charts on dashboard.
How to disable title bar from this charts?
Also font on timeline is smaller than before this commit and unreadable.
image

**Comment by murrant** [comment]
@fcqpl update your install

**Comment by murrant** [comment]
As for the font size, hmmm maybe an off by one.

**Comment by fcqpl** [comment]
> @fcqpl update your install

Thanks, this two commit fixed our issue.

https://github.com/librenms/librenms/commit/55d81fde1eee8654f3225acb3cbebd227d7f6513
https://github.com/librenms/librenms/commit/e864ed7795d80fe46c62bafbbb52f9b0e129d7a6

**Pull request merged**